js/models/languagesMd.js

```diff
@@ -6407,10 +6541,21 @@ module.exports = Backbone.Model.extend({
           retry: "Retry connection", //notTranslated 
         }       
       },
+      
       {
+        
         langName: "Polski",
         langCode: "pl",
-        /* Use capitalized keys for widely reused text that must be capitalized */
+        
+        /*
+        
+        Translator: @Mido
+        02/18/2016: ~ 71 untranslated strings
+        
+        Use capitalized keys for widely reused text that must be capitalized
```

Comment:
We should probably have these designated terms available somewhere for consistency.